stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>Minor comment but I find the quote a bit weird, I might stick with the authors having one voice so e.g. just "As we follow OWL, we knew it was possible...." rather than having a quote from a single author. Or "As our co-author Tim Powell  follows OWL, we knew it was possible..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Fixed</p>